### ISSUE TITLE
Can we push more versions to PyPi

### ISSUE THREAD

**Issue by simkessy** [agronholm/sqlacodegen#327]
### Things to check first

- [X] I have searched the existing issues and didn't find my bug already reported there

- [X] I have checked that my bug is still present in the latest release

### Sqlacodegen version

3.0.5rc5

### SQLAlchemy version

2

### RDBMS vendor

MySQL (or compatible)

### What happened?

When trying to install `pip3 install sqlacodegen==3.0.5rc5` I get an error that it's not available 

```
Looking in indexes: https://artifacts.company.com/artifactory/api/pypi/company-python-pip-prod-virtual/simple/
ERROR: Could not find a version that satisfies the requirement sqlacodegen==3.0.5rc5 (from versions: 1.0.0, 1.1.0, 1.1.1, 1.1.2, 1.1.3, 1.1.4, 1.1.5, 1.1.6, 2.0.0, 2.0.1, 2.1.0, 2.2.0, 2.3.0, 2.3.0.post1, 3.0.0b1, 3.0.0b2)
ERROR: No matching distribution found for sqlacodegen==3.0.5rc5
```

**Comment by agronholm** [comment]
There is no 3.0.5rc5 release. The latest release is 3.0.0rc5.

**Comment by simkessy** [comment]
Yes sorry that is what I mean. Personally I was able to clone it and build locally so I think I'm fine now. Feel free to close. But yea it would be easier if it was available in PyPi, but I know you're busy so no worries.

**Comment by agronholm** [comment]
I don't understand your request. I've published all releases on the PyPI. What do you think is missing?

**Comment by simkessy** [comment]
I think I'm confused but when I tried to install I got version sqlacodegen 2.3.0.post1 https://pypi.org/project/sqlacodegen/

**Comment by agronholm** [comment]
Your original post indicates that you specifically tried to install a nonexistent version.
